crates/turborepo/tests/snapshots/boundaries__boundaries_get_boundaries_lints_(npm@10.5.0).snap

```diff
@@ -0,0 +1,28 @@
+---
+source: crates/turborepo/tests/boundaries.rs
+expression: query_output
+---
+{
+  "data": {
+    "boundaries": {
+      "items": [
+        {
+          "message": "cannot import file `../../packages/another/index.jsx` because it leaves the package",
```

Comment:
Nitpick: I'd like to make a move towards writing sentences with capital letters and punctuation for our human-readable errors (including sentence fragments) when we can.

```suggestion
          "message": "Cannot import file `../../packages/another/index.jsx` because it leaves the package.",
```

Reply:
lol Rome had the opposite style guide, so still doing that. It is a little tricky when we're nesting errors cause then we can end up with multiple periods and capitals